### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by PromoFaux**
**By submitting this pull request, I confirm the following (please check boxes, eg [X]) _Failure to fill the template will close your PR_:**

***Please submit all pull requests against the `development` branch. Failure to do so will delay or deny your request***

- [x] I have read and understood the [contributors guide](https://github.com/pi-hole/pi-hole/blob/master/CONTRIBUTING.md).
- [x] I have written tests and verified that they fail without my change.
- [x] I have squashed any insignificant commits.
- [x] This change has comments for package types, values, functions, and non-obvious lines of code.
- [x] I am willing to help maintain this change if there are issues with it later.
- [x] I give this submission freely and claim no ownership. It is compatible with the EUPL 1.2 license.
- [x] I have Signed Off all commits. (`git commit --signoff`)

***Please explain what you have done and wish to accomplish with this Pull Request***

1. What does this change do, exactly?
De-duplicates dependency list (we're always forgetting to update the dep list in uninstall.sh when we add one to basic-install.sh)

This change sources the install script and uses the `distro_check` function to generate the same list as the the install script.

Also added to the output a list of each package that may have been installed by the installer, allowing the user to quickly decide if they wish to go through it and uninstall one or many (or all), or just leave them all installed.

dependencyloop

noloop

2. Please link to the relevant issues.

3. Which documentation changes (if any) need to be made because of this PR?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,6 @@
 # TestRepo
 adjustment
```
Comment (PromoFaux) [inline README.md:2]: I prefer this.

Comment (PromoFaux) [reply README.md:2]: Total review

**Review by PromoFaux — COMMENTED** [review]
Summary of review

File: README.md [hunk 2]
```diff
@@ -1,4 +1,6 @@
 # TestRepo
```
Comment (PromoFaux) [inline README.md:1]: It used to be like this

File: README.md [hunk 3]
```diff
@@ -1,4 +1,6 @@
...
 adjustment
 
-KAJSNFKJASNFKJANS
\ No newline at end of file
+KAJSNFKJASNFKJANS
+
+change for pull request
```
Comment (PromoFaux) [inline README.md:6]: test

**Review by DL6ER — DISMISSED** [review]

**Comment by DL6ER** [comment]
@PromoFaux Can you dismiss this approval?

**Review by DL6ER — APPROVED** [review]

**Review by DL6ER — APPROVED** [review]

**Review by DL6ER — DISMISSED** [review]
Requesting some changes

**Pull request closed without merging**